### DISCUSSION TITLE
Odd Rerender Behaviour with React Navigation Reset

### DISCUSSION THREAD

**Discussion by gerryfletch** [pmndrs/zustand#896] [Q&A]
Edit: No longer an issue, apologies for raising the discussion! Not due to `zustand`

**Comment by gerryfletch** [comment]
Edit: Turned out the subscription was being destroyed, but only when the reset was invoked. Makes sense. Apologies for raising the discussion

**Reply by dai-shi** [reply]
No problem! That happens.
